assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -8979,49 +8951,49 @@ private void addRemoveSubmissionsForNonElectronicAssignment(SessionState state,
         }
     }
 
-    private void initIntegrateWithGradebook(SessionState state, String siteId, String aOldTitle, String oAssociateGradebookAssignment, Assignment assignment, String title, Instant dueTime, Assignment.GradeType gradeType, String gradePoints, String addtoGradebook, String associateGradebookAssignment, String range, long category) {
+    private void initIntegrateWithGradebook(SessionState state, String aOldTitle,
```

Comment:
these method names could be better "initIntegrateWithGradebook" ;)

Reply:
It is now "configureGradebookIntegration"